docs/lua-modules/httpserver.md

```diff
@@ -42,14 +46,15 @@ Callback function has 2 arguments: `req` (request) and `res` (response). The fir
 	- `name`: Header name
 	- `value`: Header value
 
-- `ondata`: value to setup handler function HTTP data. Handler function has 2 parameters:
+- `ondata`: value to setup handler function HTTP data. Handler function has 3 parameters:
 	- `self`: `req` object
 	- `chunk`: Request data
+	- `last`: set true while the last chunk
 
 The second object holds functions:
 
-- `send(self, data, [response_code])`: Function to send data to client. `self` is `req` object, `data` is data to send and `response_code` is HTTP response code like `200` or `404` (for example)
-- `send_header(self, header_name, header_data)`: Function to send HTTP headers to client. `self` is `req` object, `header_name` is HTTP header name and `header_data` is HTTP header data for client.
-- `finish([data])`: Function to finalize connection, optionally sending data. `data` is optional data to send on connection finalizing.
+- `send(data)`: Function to send data to client. `data` is data to send if the `data` is `nil` will close the connection
+and `response_code` is HTTP response code like `200` or `404` (for example)
+- `send_header(header)`: Function to send the response header. `header` must be a table. likes `{status = 200, headers={Connection = "close"}}`. status will be set 200 if the status omitted. the built-in code is `200, 304, 404, 500`
```

Comment:
I would prefer the header array to be given separately. It is an unnecessary creation of an object which contains the status_code and the headers which don't really belong together and can as well be given separately:

- send_header(status, headers)`: Function to send the response header. status will be set 200 if the status is omitted. The built-in codes are `200, 304, 404, 500`. `headers={Connection = "close", "Content-Type" = "text/plain"}}`.

Reply:
The API I proposed above (`sendbody = res:send_headers(status, {foo = bar, ...})`) is as you describe, @HHHartmann, right?

Reply:
Ah yes the discussion is there, sorry.

Reply:
No apology needed; I was asking to make sure that the proposal matched what you wanted.  The docs still need to be updated.